.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "src/libs/redox"]
 	path = src/libs/redox
 	url = https://github.com/dcrankshaw/redox.git
+[submodule "src/libs/rapidjson"]
+	path = src/libs/rapidjson
+	url = https://github.com/giulio-zhou/rapidjson.git
```

Comment:
Main concern was that the rapidjson headers total around 15000 lines of code, which seemed a bit excessive. Otherwise there shouldn't be any problem.